features/step_definitions/journey/registration_steps.rb

```diff
@@ -21,24 +22,16 @@
 end
 
 When("I complete my registration") do
-  if @organisation_type != "charity"
-    @journey.check_your_tier_page.submit(option: :unknown)
-
+  if @organisation_type == "charity"
+    @journey.standard_page.submit
```

Comment:
Suggest a comment here:

```suggestion
    # then all tier routing questions are skipped and user is told "you need to register as a lower tier waste carrier"
    @journey.standard_page.submit
```